src/utils.js

```diff
@@ -17,12 +15,6 @@ let utils = {
     reason = reason ? reason : constants.WarningEnum.DEFAULT_TEXT;
     console.warn(constants.WarningTypes[reason]);
   },
-  generateSecretKey: function () {
```

Comment:
Due to the issues a predefined password does not work with common security standards. You saved the password in the local storege. This was not secure. Thats why I removed it. The user should use a trusted endpoint to deliver the password. In the most cases this would be the logged in user or a trusted third party.